backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -146,7 +144,7 @@ export class AdminResolver {
     }
     // moderator user disabled own account?
     const moderatorUser = context.user
```

Comment:
cant we use `getUser` here aswell? We should not be able to not have a user on this call anyways - so the `getUser()` function should never throw.